src/main/webapp/dashboards/admin.jsp

```diff
@@ -15,144 +15,161 @@
     <title>Admin Page</title>
 </head>
 <body>
-    <h1>SUPER SECRET ADMIN STUFF</h1>
-    <h2>Audit Log</h2>
-    <ol style="list-style-type: none">
-        <c:forEach items="${requestScope.events}" var="event">
-            <li>${event.createdAt} INFO ${event.description}</li>
-        </c:forEach>
-    </ol>
 
-    <h3>User Account List</h3>
-    <ol style="list-style-type: none">
-        <c:forEach items="${requestScope.users}" var="userAccount">
-            <c:if test="${userAccount.active}">
-                <li>${userAccount.firstName} ${userAccount.surname}</li>
+<div class="outer">
+    <div class="header">
+        <div class="innerDashboard">
+            <img src="${pageContext.request.contextPath}/images/temp_logo.png" alt="logo" id="header-logo">
+            <ul>
+                <li><a href="#">Log Out</a></li>
```

Comment:
Should this be made to work in this PR? It would be a matter of surrounding it in a form that POSTs /login (same as the other button on the page)

Reply:
Not it shouldn't. This was simply to style the navigation bar so that people can replicate it on other pages. Whoever is doing the full CSS for this page would do this.